tests/SelfTest/IntrospectiveTests/Sharding.tests.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Distributed under the Boost Software License, Version 1.0. (See accompanying
+ *  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+ */
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/generators/catch_generators_all.hpp>
+
+#include <catch2/internal/catch_sharding.hpp>
+
+#include <unordered_map>
+#include <vector>
+
+TEST_CASE("Sharding Function") {
```

Comment:
```suggestion
TEST_CASE("Sharding Function", "[approvals]") {
```

The approval tests are getting too big, I will be cutting them down a bit later, but for now I don't think these need to be included.